src/Tribe/Shortcode/Shortcode_Abstract.php

```diff
@@ -31,7 +31,16 @@ abstract class Shortcode_Abstract implements Shortcode_Interface {
 	protected $default_arguments = [];
 
 	/**
-	 * Array of callbacks for arguments validation
+	 * Account for this shortcode having aliased arguments.
+	 *
+	 * The array keys are aliases of the array values (i.e. the "real" shortcode attributes to allow parsing).
+	 *
+	 * @var string[]
+	 */
+	public $aliased_arguments = [];
```

Comment:
We should move this to a protected prop, and add a `set_*` and `get_*` into this.

Reply:
make sure the `set_*` and `get_*` are on the interface.

Reply:
done